### PR TITLE
Fix readme

### LINKED ISSUE #47: Readme file needed for this module
This should contain the version of ksdk used, what changes are there and provide useful information about this module

### PR DESCRIPTION
**Pull request by 0xc0170**
Should fix #47. 

I removed txt file, and replaced it with readme.

@iriark01 @bogdanm

### CODE REVIEW

**Comment by bogdanm** [comment]
+1

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# mbed HAL ksdk mcu
```
Comment (iriark01) [inline readme.md]: mbed HAL KSDK MCU

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+This is the HAL implementation for mcu which are supported by KSDK. Part of this module is [Freescale KSDK](http://www.freescale.com/tools/software-and-tools/run-time-software/kinetis-software-and-tools/development-platforms-with-mbed/software-development-kit-for-kinetis-mcus:KINETIS-SDK) version 1.0.0.
```
Comment (iriark01) [inline readme.md]: ... for MCUs that are supported by KSDK.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+## Freescale KSDK code
+
+KSDK code is located in under TARGET_KSDK_CODE. We had to do couple of modifications, to be able to use it in mbed.
```
Comment (iriark01) [inline readme.md]: KSDK code is located under TARGET_KSDK_CODE. We had to do a couple of modifications to be able to use it in mbed OS.

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+
+- this module does not contain entire KSDK. Only few drivers and HAL are included.
```
Comment (iriark01) [inline readme.md]: - This module does not contain the entire KSDK. Only a few drivers and HAL are included.

File: readme.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+- a header file like fsl_peripheral_features (peripheral here is any freescale mcu peripheral, lpuart, dspi, ..) defines features for MCU. In original KSDK features files, it would display an error (No valid CPU defined), thus we defined MBED_NO_PERIPHERAL and removed the error.
```
Comment (iriark01) [inline readme.md]: - A header file, like fsl_peripheral_features (peripheral here meaning any Freescale MCU peripheral like LPUART and DSPI), defines features for the MCU. In the original KSDK feature files it displayed an error ("No valid CPU defined), so we defined MBED_NO_PERIPHERAL and removed the error.

**Comment by 0xc0170** [comment]
@iriark01 All addressed, thanks

**Pull request merged**